src/lib/support/jsontlv/JsonToTlv.cpp

```diff
@@ -0,0 +1,418 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <algorithm>
+#include <json/json.h>
+#include <lib/support/Base64.h>
+#include <lib/support/SafeInt.h>
+#include <lib/support/jsontlv/ElementTypes.h>
+#include <lib/support/jsontlv/JsonToTlv.h>
+
+namespace chip {
+
+namespace {
+
+std::vector<std::string> splitIntoFieldsBySeparator(const std::string & input, char separator)
```

Comment:
```suggestion
std::vector<std::string> SplitIntoFieldsBySeparator(const std::string & input, char separator)
```

Reply:
This issue is still unresolved: https://github.com/project-chip/connectedhomeip/pull/27635#discussion_r1275428736

Reply:
updated